src/App.jsx

```diff
@@ -14,33 +16,144 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const buttonsFromServer = [
+  'Sort alphabetically',
+  'Sort by length',
+  'Reverse',
+  'Reset',
+];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+function getPreparedButtons(buttons, { sortField, reverse }) {
+  const preparedButtons = [...buttons];
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (!sortField && !reverse) {
+    preparedButtons.pop();
+  }
+
+  return preparedButtons;
+}
```

Comment:
Consider renaming `getPreparedButtons` to start with a verb, such as `prepareButtons`, to align with naming conventions and improve readability.